device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceTwin/Property.java

```diff
@@ -23,13 +23,6 @@ public Property(String key, Object value)
              */
             throw new IllegalArgumentException("Key cannot be null or empty");
         }
-        if (key.contains(" ") || key.contains("$") || key.contains("."))
-        {
-            /*
-            **Codes_SRS_Property_25_006: [**If the key contains illegal unicode control characters i.e ' ', '.', '$', the constructor shall throw an IllegalArgumentException.**]**
-             */
-            throw new IllegalArgumentException("Key cannot contain illegal unicode control characters '.', '$', ' '");
```

Comment:
Can you split this commit into separate one. Please follow github hygiene for meaningful commits.

Reply:
Sure, now this PR is just about changing artifact id's to add "-preview" suffix